diff -rupN tinyos-tools-1.4.0/platforms/msp430/pybsl/Makefile.am tinyos-tools-1.4.0-new/platforms/msp430/pybsl/Makefile.am
--- tinyos-tools-1.4.0/platforms/msp430/pybsl/Makefile.am	2006-07-12 19:00:39.000000000 +0200
+++ tinyos-tools-1.4.0-new/platforms/msp430/pybsl/Makefile.am	2011-10-25 19:34:02.100745600 +0200
@@ -11,7 +11,7 @@ if CYGWIN
 # (this code is not compatible with cygwin's python)
 
 install: tos-bsl-win.tar.gz
-	tar xzvf tos-bsl-win.tar.gz -C $(bindir) 
+	tar xzvf tos-bsl-win.tar.gz -C $(DESTDIR)$(bindir) 
 
 else
 # Linux support
